node/dataSources/fixedPrice.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  RequestOptions,
+  RESTDataSource
+} from 'apollo-datasource-rest'
+import { forEachObjIndexed } from 'ramda'
+
+export class FixedPriceDataSource extends RESTDataSource<Context> {
```

Comment:
this should not be a `FixedPrice` data source, but a `PricingDataSource`. Fixed is just one possible API, more apis may be added on the future.

Reply:
Fixed